src/main/java/org/broadinstitute/hellbender/utils/logging/DoFnWLog.java

```diff
@@ -1,9 +1,8 @@
-package org.broadinstitute.hellbender.dev;
+package org.broadinstitute.hellbender.utils.logging;
```

Comment:
Could you move this class to `engine/dataflow` (with the rest of the engine-level DoFns/transforms) -- we want to try to keep all dataflow code together for now